cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PullReplication.java

```diff
@@ -33,8 +33,8 @@
     /**
      * URI for this replication's remote database.
      *
-     * Include username and password in the URL, or supply an Authorization header using
-     * setCustomHeaders() in CouchConfig.
+     * Include username and password in the URL, or supply an Authorization header using an
+     * HttpConnectionRequestInterceptor
```

Comment:
Nope, `CookieInterceptor` is available by default, but we don't replace the creds yet, the plan is to add that in the future, not sure when it is planned for though